privateSBOMExchange/tests/test_cpabe_AES.py

```diff
@@ -0,0 +1,68 @@
+import copy
+from lib4sbom.parser import SBOMParser
+import json
+import argparse
+
+from petra.lib.models.tree_ops import build_sbom_tree, verify_sameness
+from petra.lib.models import MerkleVisitor, EncryptVisitor, DecryptVisitor
+from petra.lib.util.config import Config
+
+import cpabe
+
+argparser = argparse.ArgumentParser()
+# TODO: add args for the config
+# TODO: handle defaults etc
+argparser.add_argument("-o", "--original-file", type=str, required=True, help="the file to which to write the original SBOM tree")
+argparser.add_argument("-r", "--redacted-file", type=str, required=True, help="the file to which to write the redacted SBOM tree")
+argparser.add_argument("-d", "--decrypted-file", type=str, required=True, help="the file to which to write the decrypted SBOM tree")
```

Comment:
I'm not sure if `-o` is overwriting the source file?

Reply:
oh, the flag was confusing for me. Nvm

Reply:
I'm also happy to rename the flag to something like `-p` or `-u` for pre- or un-redacted.